prow/jobs/test-infra/configs.jobs.yaml

```diff
@@ -0,0 +1,16 @@
+presubmits:
+  kyma-project/test-infra:
+  - name: prow/test-infra/check_config
+    trigger: "(?m)^/check config"
+    rerun_command: "/check config"
+    run_if_changed: "prow/(plugins|config).yaml"
+    decorate: true
+    skip_report: true
```

Comment:
For `test-infra` reporting should be enabled.

Reply:
Reporting has been enabled